src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## March 28, 2023
+
+### iOS Core 3.8.2
+
+* Fixed some crashes which can happen on shutdown.
```

Comment:
Missing the 3.8.1 note here.

Reply:
The 3.8.1 note was already merged. It's down below.